nin/backend/nin

```diff
@@ -63,6 +64,34 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    if(!which('nin')) {
+      echo('Can\'t find nin executable in PATH/PATHEXT');
+      echo('Make sure \'/path/to/nin/nin/backend\' is present.');
+      exit(0);
+    }
+    echo('Checking for new version of nin');
+
+    var ninExecutable = which('nin');
+    var ninPath = ninExecutable.split('/');
+    var ninPath = ninPath.splice(0, ninPath.length-3).join('/');
```

Comment:
You can remove everything above this line and replace it with `__dirname`,
as this points to the path of the executable folder.
See https://github.com/ninjadev/nin/blob/master/nin/backend/compress.js#L57 for an example.